src/building/bootstrapping.md

```diff
@@ -78,6 +78,47 @@ contribution [here][bootstrap-build].
 
 ## Stages of bootstrap
 
+Like most other bootstrapping compilers, `rustc` is compiled in stages.
+_Unlike_ most other compilers, where `stage0` refers to the bootstrap compiler,
```

Comment:
This is .. well, not *quite* true. stage0/bin/rustc is the bootstrap compiler. Really, we should rename it, but "to what?" then the question is.